config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -278,6 +200,42 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
+- cron: '1 8 * * 6' # Run at 00:01PST (8:01 UTC) on Sat
```

Comment:
Can you move it after scale-correctness (for alphabetical order)?

Reply:
Done.